scripts/get_weather.py

```diff
@@ -0,0 +1,24 @@
+import requests, json
```

Comment:
You're still missing the shebang line that goes before the import statement `#!/usr/bin/python3`